### PR TITLE
Extend the max message size, from default to 10mb.

### PR DESCRIPTION
**Pull request by morrowc**
gRPC has a max message size of 4mb by default.
Extend that to avoid some of the larger update messages causing:
  ResourceExhausted desc = grpc: received message larger than max (4944783 vs. 4194304)

### CODE REVIEW

**Comment by morrowc** [comment]
+yuhaoh-he for a review :)

File: cmd/archive_upload_client/client.go [hunk 1]
```diff
@@ -22,6 +22,12 @@ import (
 	grpcMetadata "google.golang.org/grpc/metadata"
 )
 
+const (
+	// Max message size set to 10mb.
```
Comment (yuhao-he) [inline cmd/archive_upload_client/client.go:26]: Duplicate comment?

File: cmd/archive_upload_client/client.go [hunk 2]
```diff
@@ -22,6 +22,12 @@ import (
...
+	// Max message size set to 10mb.
+	maxMsgSize = 10 * 1024 * 1024
```
Comment (yuhao-he) [inline cmd/archive_upload_client/client.go:28]: RouteViews has some update archives over 10 MiB: http://routeviews.org/route-views3/bgpdata/2021.12/UPDATES/ 

Maybe 50 is a good line?

Another thing: will we ever consider to import RV RIBs? Those can be pretty large (i.e. over 100 MiBs), but I guess we can worry about those later.

File: cmd/archive_upload_client/client.go (hunk 1 above)
Comment (morrowc) [reply cmd/archive_upload_client/client.go:26]: fixed.

File: cmd/archive_upload_client/client.go (hunk 2 above)
Comment (morrowc) [reply cmd/archive_upload_client/client.go:28]: I think setting to 50m sounds fine, we can cross the RIB part later yea.

**Review by yuhao-he — APPROVED** [review]

**Pull request merged**